gef-ui/templates/includes/application-preview/task-comments.njk

```diff
@@ -1,43 +1,51 @@
-            {% else %}
-              <strong>You are proceeding with UKEF cover and accepting the following conditions:</strong>
-            {% endif %}
-            <p>{{ ukefDecision[0].text | safe }}</p>
+    {% if renderReviewDecisionLink %}
+      <div class="govuk-!-margin-bottom-6" data-cy="ukef-review">
+        <a href="/gef/application-details/{{ dealId }}/review-decision" class="govuk-link">Review UKEF decision</a>
+      </div>
+    {% endif %}
+    {% if coverDatesConfirmed %}
+      {% if ukefDecision[0].decision == 'Accepted (without conditions)' %}
+        <strong>You are proceeding with UKEF cover</strong>
+      {% elif ukefDecision[0].decision == 'Accepted (with conditions)' %}
+        {% if applicationStatus == "Ready for Checker\'s approval" %}
+          <strong>{{ application.maker.firstname }}
+            {{ application.maker.surname }} has accepted the following conditions</strong>
+        {% else %}
+          <strong>You are proceeding with UKEF cover and accepting the following conditions:</strong>
         {% endif %}
+        <p>{{ ukefDecision[0].text | safe }}</p>
       {% endif %}
-
-      <div data-cy="latest-comment">
-        {% for comment in comments %}
-          <strong>Comments from {{ comment.role }} ({{ comment.userName }})</strong>
-          <p>{{ comment.createdAt | localiseTimestamp('DD MMM YYYY h:mma', timezone) }}</p>
-          <p>{{ comment.comment | safe }}</p>
-        {% endfor %}
+    {% endif %}
+    {% if unissuedFacilitiesPresent and facilitiesChangedToIssued.length === 0 %}
+      <h2 data-cy="update-unissued-header">Update facility stage for unissued facilities</h2>
+      <div class="govuk-!-margin-bottom-6">
+        <a href="/gef/application-details/{{ dealId }}/unissued-facilities" class="govuk-link" data-cy="update-unissued-link">
+          View unissued facilities</a>
       </div>
-
-    {% endset %}
-
-    {{ mojBanner({
+    {% endif %}
+    {% if facilitiesChangedToIssued.length > 0 %}
+      <h2 data-cy="update-unissued-stage">Review facility stage</h2>
+      <h3 data-cy="changed-unissued-header">The following facility stages have been updated to issued:</h3>
+      <div class="govuk-!-margin-bottom-6" data-cy="unissued-to-issued-facilities-list">
+        {% for item in facilitiesChangedToIssued %}
+          <p>{{ item.name }}</p>
+        {% endfor %}
+      </ul>
+    </div>
+  {% endif %}
+  <div data-cy="latest-comment">
+    {% for comment in comments %}
+      <strong>Comments from {{ comment.role }} ({{ comment.userName }})</strong>
+      <p>{{ comment.createdAt | localiseTimestamp('DD MMM YYYY h:mma', timezone) }}</p>
+      <p>{{ comment.comment | safe }}</p>
+    {% endfor %}
+  </div>
+  {% endset %}
+  {{ mojBanner({
       type: 'no-icon',
       html: bannerHtml
-    }) }}
-
-  </header>
-{% endif %}
+    }) }}</header>{% endif %}
```

Comment:
Can we re-add the spaces in this file - not sure how anyone else feels but I find it really hard to read when there's no spaces between complex NJK loops/conditions

Reply:
done - my editor was removing the spaces